iset.mm

```diff
@@ -19034,9 +19036,15 @@ This law is thought to have originated with Aristotle (_Metaphysics_,
   $}
 
   ${
-    syl5eq.1 $e |- A = B $.
-    syl5eq.2 $e |- ( ph -> B = C ) $.
-    $( An equality transitivity deduction.  (Contributed by NM, 5-Aug-1993.) $)
+    eqtrid.1 $e |- A = B $.
+    eqtrid.2 $e |- ( ph -> B = C ) $.
+    $( An equality transitivity deduction.  (Contributed by NM,
+       21-Jun-1993.) $)
+    eqtrid $p |- ( ph -> A = C ) $=
+      ( wceq a1i eqtrd ) ABCDBCGAEHFI $.
+
+    $( Renamed to ~ eqtrid .  Kept during a transition period.  DO NOT USE.
+       (Contributed by NM, 21-Jun-1993.) $)
```

Comment:
When do you plan to repace syl5eq by eqtrid in iset.mm (and then to remove syl5eq afterwards)?

Reply:
For most of the renames I have been doing set.mm and iset.mm together. So if @wlammen wants to do eqtrid as part of the existing pull request series, that is fine with me.

The only reason I got the rename started in iset.mm in this pull request is that I was copy pasting an eqtrid reference from set.mm to iset.mm.

Reply:
I'll get my notebook back by tomorrow, I think.  About then I'm going to resume #4320.  Don't hesitate to continue this task if It takes too long.